app/pages/resources/contact.tsx

```diff
@@ -21,19 +23,28 @@ class Contact extends Component<Props> {
     }
   `;
 
-  // TODO: Add content to this empty page
   render() {
     const {query} = this.props;
     const {session} = query || {};
     return (
       <DefaultLayout session={session} title="Contact Us">
         <Container>
-          <p>
-            Please email{' '}
-            <a href="mailto:GHGRegulator@gov.bc.ca?subject=CIIP Portal Inquiry">
-              GHGRegulator@gov.bc.ca
-            </a>{' '}
-            with any questions or concerns.
+          <Alert variant="info">
+            <FontAwesomeIcon icon={faInfoCircle} />
+            <span className="pl-2">
+              For help with your CIIP application, or questions about the
+              CleanBC Industrial Incentive Program, please email{' '}
+              <a href="mailto:GHGRegulator@gov.bc.ca?subject=Support Request (CIIP)">
+                GHGRegulator@gov.bc.ca
```

Comment:
We've been trying to replace any hardcoded constants like contact email with getting them from a runtime variable. 

Those are configured in [`values.yaml`](https://github.com/bcgov/cas-ciip-portal/blob/b37f88784871d92f7dbdc0679888a9e2b6b2c8a7/helm/cas-ciip-portal/values.yaml#L63-L64) (as well as in the corresponding `values-[env].yaml` if we want to override that in any particular env), and set in here the Next runtime in [`next.config.js`](https://github.com/bcgov/cas-ciip-portal/blob/b37f88784871d92f7dbdc0679888a9e2b6b2c8a7/app/next.config.js#L25). Usage example [here](https://github.com/bcgov/cas-ciip-portal/blob/b37f88784871d92f7dbdc0679888a9e2b6b2c8a7/app/components/helpers/HelpButton.tsx#L17)

It looks like `adminEmail` (for GHGRegulator) still needs to be set up in the Next config for use on the client side, even though we've done all the variable replacements for `SUPPORT_EMAIL`. 

This might be a good opportunity to remove any other hardcoded client-side instances of the admin email at the same time.

Reply:
Pulling in this ticket to address the issues: https://youtrack.button.is/issue/GGIRCS-2015